Arriba/Arriba.Test/Model/TableTestsLarge.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+using Arriba.Extensions;
+using Arriba.Model;
+using Arriba.Model.Aggregations;
+using Arriba.Model.Column;
+using Arriba.Model.Expressions;
+using Arriba.Model.Query;
+using Arriba.Serialization;
+using Arriba.Structures;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Arriba.Test.Model
+{
+    [TestClass]
+    public class TableTestsLarge
+    {
+        [TestCleanup]
```

Comment:
Can we have a shared TestInit which makes the table (or a static member which builds it only the first time)? I want to avoid having multiple tests build it.